Day-01/zig/emmaboecker/day01.zig

```diff
@@ -0,0 +1,72 @@
+const std = @import("std");
+
+pub fn main() !void {
+    const input = @embedFile("./input/day01.txt");
```

Comment:
Please place the input file in the same folder as the code

```suggestion
    const input = @embedFile("./day01.txt");
```